shpec/shpec_shpec.sh

```diff
@@ -64,16 +65,25 @@ line'
 
   describe "passing through to the test builtin"
     it "asserts an arbitrary algebraic test"
-      assert test "[[ 5 -lt 10 ]]"
+      assert test "[ 5 -lt 10 ]"
     end
   end
 
   describe "stubbing commands"
+    # only bash lets us redefine 'exit', so use 'false'
```

Comment:
ce61f728abd138ae1b8e61a57fba048e4b84b15b contains the reasoning behind this test. My preference is to leave that kind of thing in git, and not in the code itself.

Additionally, I would say that this initial test is unnecessary; the following two tests cover both the stubbed functionality and the original functionality. If any part of that stub/unstub process fails, one of these commands will indicate so.